libexec/erlang

```diff
@@ -201,14 +201,6 @@ erlang_generate_release() {
   __exec_if_defined "pre_erlang_generate_release"
 
   if [ "$RELEASE_CMD" = "mix" ]; then
-    [[ "$USING_DISTILLERY" != "true" ]] && __detect_exrm_version
-    [[ "$USING_DISTILLERY" != "true" ]] && [[ "$EXRM_VERSION_MAJOR" -gt "0" || "$EXRM_VERSION_MINOR" -gt "19" \
-       || "${EXRM_VERSION_MAJOR}.${EXRM_VERSION_MINOR}.${EXRM_VERSION_PATCH}" = "0.19.9" ]] \
-    && local _force_no_interaction=" --no-confirm-missing"
-    # exrm normal verbosity is very silent, so use --verbosity=normal for the release task if edeliver runs
-    # in verbose mode. if --verbosity=verbose is used for the exrm release task, the output is very very verbose,
-    # so we use it only if edeliver runs in silent mode because the very very verbose output is not visible
-    # until the release task fails and then it might be helpful to see the very very verbose output.
     if [[ "$USING_DISTILLERY" = "true" ]]; then
```

Comment:
If I understand this correctly, this should always evaluate to true because we are only using distillery.